.github/ci/packages-jammy.apt

```diff
@@ -0,0 +1,7 @@
+libdart-collision-ode-dev
+libdart-dev
+libdart-external-ikfast-dev
+libdart-external-odelcpsolver-dev
+libdart-utils-urdf-dev
+libogre-next-dev
+python3-ignition-math6
```

Comment:
Should this be math7?

Reply:
yes! Thanks!

Reply:
[e399431](https://github.com/ignitionrobotics/ign-gazebo/pull/1446/commits/e399431488445098a8b889021fa6a0705ad4a403)